.github/workflows/scala-steward.yml

```diff
@@ -1,4 +1,4 @@
-name: Launch Scala Steward
+name: Dependencies
```

Comment:
do we now have two workflows with the same name "Dependencies"? (this one and dependency-graph.yml)

Reply:
Yes, because they both relate to dependencies.